src/components/audio-settings/volume-slider.tsx

```diff
@@ -0,0 +1,36 @@
+import type { FC } from "react";
+import { Box, Button, Slider, Stack } from "@mui/material";
+import { VolumeDown, VolumeUp } from "@mui/icons-material";
+import { AUDIO_OFF, AUDIO_ON, DEFAULT_VOLUME } from "~/constants/index.js";
+
+interface Props {
+  volume: number;
+  changeVolume: (volume: number) => void;
+}
+export const VolumeSlider: FC<Props> = ({ volume, changeVolume }) => {
+  const handleChange = (event: Event, newValue: number | number[]) => {
+    changeVolume(newValue as number);
+  };
```

Comment:
Don't cast an array to a number. Use `if (typeof newValue === "number") {} else {}` to handle both cases